src/wp-includes/script-loader.php

```diff
@@ -1220,6 +1221,18 @@ function wp_default_scripts( $scripts ) {
 			'publishSettings'         => __( 'Publish Settings' ),
 			'invalidDate'             => __( 'Invalid date.' ),
 			'invalidValue'            => __( 'Invalid value.' ),
+			'blockThemeNotification'  => sprintf(
+				/* translators: 1. %s: URL to the Site Editor admin screen, 2: "Switch to Site Editor" button placeholder. */
+				__( 'To get the best customization experience from your block theme, use the Site Editor. <a href="%1$s" target="_blank">Tell me more</a>. %2$s' ),
+				'https://wordpress.org/support/article/site-editor/',
```

Comment:
This link need to be translatable as some locales translated this page.

Reply:
Fixed in e1e867c9b0405e865c174375df3e6debdffcc4d2

Reply:
Thanks!